packages/radio-button/radio-button.scss

```diff
@@ -1,5 +1,4 @@
-@import "~@fremtind/jkl-core/build/scss/typography/font-size.scss";
-@import "~@fremtind/jkl-core/build/scss/variables/colors.scss";
+@import "~@fremtind/jkl-core/variables/variables";
```

Comment:
Not a huge fan of the semantic `/variables/variables`.

Doesn't _index.scss work here? So you can do `@import "~@fremtind/jkl-core/variables";`

Reply:
Maybe? I agree that it would be better, so I'll check it out!

Reply:
Seems like this was only implemented in LibSass in May, and is not a part of node-sass yet. Should be right around the corner, though!

https://github.com/sass/node-sass/issues/2339

Reply:
okay, how about all.scss?

import variables/all.scss?